help/faq/statfaq.md

```diff
@@ -1,82 +1,14 @@
-    2nd requests for a viewable version, perhaps artificially doubling
-    the initial raw number of requests.
-    
-    In addition, it is impossible to distinguish between actual reading
-    and browsing or other spurious requests. For example, many
-    physicists tend to use these papers as a substitute for memory --
-    i.e. they know a paper with a particular equation, so get it just
-    for that equation then delete it (should that count as reading?).
-    
-    *So before publicizing any such statistics, all the caveats would
-    have to be clearly understood.*
-    
-    But then the very availability of statistics could have many
-    undesirable consequences. Some segment of submitters might find it
-    highly embarrassing to see how little read is (and consequently how
-    little impact has) their magnum opus on which they've expended so
-    much quality time and effort. Furthermore, we don't wish to
-    encourage people to try somehow to distort public statistics in
-    their favor either by self-generating large numbers of requests, or
-    by repeated replacement of their submission to generate multiple
-    repeat requests.
-    
-    It could be argued perhaps correctly that statistics may provide
-    some useful information at least on the *relative* popularity of
-    submissions, since the distributed access and other factors may be
-    subsumable into some overall scale factor. But even this information
-    is ambiguous in many cases, and publicizing, even when accurate,
-    could merely accentuate faddishness in fields already excessively
-    faddish.
-    
-    *Most significantly*, however, there is a strong philosophic reason
-    for not publicizing (or even saving) these statistics. When one
-    browses in a library it is very important (in fact legislated) that
-    big brother is not watching through a camera mounted on the wall;
-    for the benefit of readers it is very important to maintain in every
-    way possible this sense of freedom from monitoring in the electronic
-    realm.
+  - ### Why?
 
-  - ### But it used to be that I could receive preprint/reprint requests by mail, and thereby identify people with similar interests. Aren't I losing something?
-    
-    No.
-    
-    You can add explicitly at the top of your paper something like:
+    arXiv has a strong philosophic reason for not publicizing these statistics.
     
-    > "If you have read this paper and wish to be included in a mailing
-    > list (or other means of communication) that I maintain on the
-    > subject, then send e-mail To: me@my.email.address ."
-    
-    A few moments' reflection will convince you that this is a much more
-    effective means of identifying parties truly interested in your
-    research, as opposed to random passersby.
+  - ## Do you plan to use download statistics in any way?
 
-  - ### Do you plan to use download statistics in any way?
-    
     Yes.
+

```

Comment:
Yes, we track and report the number of [downloads from arXiv](help/stats) and for our [institutional membership](/about/membership), but we do not track by individual papers.

Reply:
@jlmorano take a look at this pull https://github.com/arXiv/arxiv-docs/pull/123/commits/114b7b46e7defb0d5565c06d9e5b919fc08f1084 I think it's what we're looking for now.

Reply:
(I've already fixed the extra comma in https://github.com/arXiv/arxiv-docs/pull/123/commits/1e67cc3d08c5a7d5ac520138d25cee82680628df, so that shouldn't be an issue)